### DISCUSSION TITLE
How can I get the Content rendered data?

### DISCUSSION THREAD

**Discussion by bdkoder** [elementor/elementor#25082] [Developers Community]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

How can I get the Content rendered data?

https://prnt.sc/yyxLhvdjMALV

### Steps to reproduce

NA

### Expected behavior

NA

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
NA
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @bdkoder,

please don't use Elementor GitHub issues to ask questions about the developer API. GitHub Issues are reserved only for Elementor Bug Reports.

If you want to learn about the Elementor Developer Documentation, please visit the [Developers Center](https://developers.elementor.com).

Regards

**Reply by bdkoder** [reply]
Thank you, but how can I get the solution?

**Comment by bdkoder** [comment]
It's not working - $elementor_content = \Elementor\Plugin::instance()->frontend->get_builder_content_for_display( $post_id );

**Reply by bdkoder** [reply]
@nicholaszein , Would you please look into this matter?
